wallet/how-to/get-started-building/run-devnet.md

```diff
@@ -5,46 +5,84 @@ sidebar_position: 2
 
 # Run a development network
 
-You can run a personal Ethereum development network using [Ganache](https://www.trufflesuite.com/ganache),
+You can run a personal Ethereum development network using [Hardhat](https://hardhat.org/),
 which allows you to develop a dapp in a secure test environment.
 
 :::note
-When using a local development blockchain such as Ganache or
+When using a local development blockchain such as Hardhat or
```

Comment:
"HardHat Network", is the HardHat component that runs the dev networks. So not sure whether we want to use the component name or platform name. 
https://hardhat.org/hardhat-network/docs/overview#hardhat-network